selectors/i18n.test.js

```diff
@@ -25,7 +25,7 @@ describe('selectors/i18n', () => {
             expect(getCurrentLocale(state)).toEqual('fr');
         });
 
-        test('not logged in', () => {
+        test('logged in', () => {
```

Comment:
CMIIW, I assume this was an unintended typo?

Reply:
Yep, that looks like an incorrect copy-paste on my part. Good catch